Extensions/Inhouse/Source/Models/InhouseQueue.cs

```diff
@@ -9,25 +9,28 @@ namespace Discord.Inhouse
 {
     public enum Rank
     {
-        GrandChamp = 1500,
-        Champ3 = 1400,
-        Champ2 = 1300,
-        Champ1 = 1200,
-        Diamond3 = 1100,
-        Diamond2 = 1000,
-        Diamond1 = 930,
+        SuperSonicLegend = 1880,
```

Comment:
nit: I think Supersonic is one word ?
```suggestion
        SupersonicLegend = 1880,
```

Reply:
resolved